modules/util/slice.go

```diff
@@ -53,3 +53,11 @@ func Sorted[S ~[]E, E cmp.Ordered](values S) S {
 	slices.Sort(values)
 	return values
 }
+
+func ValuesOfMap[K comparable, V any](m map[K]V) []V {
+	values := make([]V, 0, len(m))
+	for _, v := range m {
+		values = append(values, v)
+	}
+	return values
+}
```

Comment:
Why not `maps.Values`?

Reply:
edf7a7801c5a4af126e5b958dae1cdb02e49144b